packages/amplify/amplify_flutter/example/linux/my_application.cc

```diff
@@ -1,3 +1,6 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
They're added to all the platforms' entrypoint files - so every platform's "main" since out of all the generated boilerplate, we do occasionally update these